README.md

```diff
@@ -4,27 +4,55 @@ This repository contains all relevant project necessary to have [polkadot](https
 
 This is WIP and will evolve frequently.
 
-## How to use `pallet-chainlink`?
-
-Complete documentation is accessible in the pallet [README](pallet-chainlink/README.md).
+## What is Polkadot and Chainlink?
 
-See the full [example](substrate-node-example/runtime/src/example.rs) for more details.
+[Chainlink](https://chain.link/) is a decentralized oracle technology that allows off-chain data to be connected to any chain.
+[Polkadot](https://polkadot.network/) is a scalable, heterogeneous, multi-chain technology, that can be used to create, maintain, and connect blockchains.
 
 ## Run the example
 
+### Requirements
+
+[Rust](https://www.rust-lang.org/tools/install)
+[Docker](https://docs.docker.com/get-docker/)
+[jq](https://stedolan.github.io/jq/download/)
+
 `rust-toolchain` is used to make sure the correct rust version is used. Make sure to install the WASM target using:
 
 ```
 rustup target add wasm32-unknown-unknown
 ```
 
-`substrate-node-example` shows off out to use `pallet-chainlink` end-to-end.
+### Running the example
+
+[You can view a youtube demo here.](https://www.youtube.com/watch?v=0rZghy0TIOQ&feature=emb_title)
+
+`substrate-node-example` shows off out to use `pallet-chainlink` end-to-end. It first makes a local dummy substrate built parachain, and then spins up some local Chainlink nodes that have parameters to connect to the parachain.
 To test:
 
-* start the chain using `make run-chain`
-* start the frontend using `make run-front-end`
+- start the chain using `make run-chain`
+- start the frontend using `make run-front-end`
+
+To spin up some simple Chainlink nodes to intereact with the parachain you just created, run the setupcommand:
+`cd substrate-chainlink`
+`./setup`
+
+This will run a number of docker commands that will spin up your Chainlink node. You can access them by going to your browser at:
+`http://localhost:6691/`
+You can see in the logs of `./setup` if the address has moved.
+
+_note_
+If you'd like to restart you will have to remove all the docker containers. You can do this by running:
```

Comment:
What do you mean by restart?

You should be able to do `docker-compose down` to stop everything and `docker-compose up` to get back to where you were.

To reset everything, you need to run what's below.

Just to clarify, the `./setup` script is a one-time initialization script. You only have to run the docker containers to run the demo.

Reply:
Ah ok! This is good information. I'll update the PR. thanks for your edits!